### ISSUE TITLE
Audit Forms for Accessibility 

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#50055]
Perform a quick Adobe based accessibility scan/audit of them all in 2-3 days. And I'd could rank them as no errors; simple errors (pdf just needs to be slightly edited); complex errors (pdf should be completely recreated from scratch)
